packages/tools/kadena-cli/src/networks/commands/networkDelete.ts

```diff
@@ -12,19 +24,34 @@ export const deleteNetworksCommand: (
 ) => void = createCommand(
   'delete',
   'Delete local network',
-  [
-    globalOptions.network({ isOptional: false }),
-    networkOptions.networkDelete(),
-  ],
+  [globalOptions.networkSelect(), networkOptions.networkDelete()],
   async (option) => {
-    const networkData = await option.network();
-    const deleteNetwork = await option.networkDelete();
+    const defaultNetworkName = await getDefaultNetworkName();
+    const networkData = await option.network({
+      default: defaultNetworkName,
+    });
+
+    networkData.network = networkData.network ?? defaultNetworkName;
```

Comment:
you're already passing `defaultNetworkName` to `option.network()` and here still using it again to set the default, what is it used for in the option then?

Reply:
missed to remove this! will update it